generated/alerting/client.go

```diff
@@ -414,6 +414,11 @@ func (c *APIClient) prepareRequest(
 			localVarRequest.SetBasicAuth(auth.UserName, auth.Password)
 		}
 
+		// APIKey Authentication
```

Comment:
This has been manually added to the generated client? I think that's fine for now, these openapi clients are a bit of a mess right now. We need to tidy this up. 

Can we add a comment calling out that this is inserted after generation.